easybuild/tools/module_generator.py

```diff
@@ -146,14 +186,17 @@ def comment(self, msg):
         """Return string containing given message as a comment."""
         return "# %s\n" % msg
 
-    def conditional_statement(self, condition, body, negative=False):
+    def conditional_statement(self, condition, body, negative=False, indent_body=True):
         """Return formatted conditional statement, with given condition and body."""
         if negative:
             lines = ["if { ![ %s ] } {" % condition]
         else:
             lines = ["if { [ %s ] } {" % condition]
 
-        lines.append('    ' + body)
+        if indent_body:
+            body = '    ' + body
```

Comment:
can we use a class constant `INDENTATION` or at least `' ' * 4` so we can read this?